tools/flect/initializer.go

```diff
@@ -0,0 +1,57 @@
+package flect
+
+import (
+	"context"
+	"fmt"
+	"os"
+)
+
+type Initializer struct{}
+
+func (i Initializer) Name() string {
+	return "flect/initializer"
+}
+
+func (i *Initializer) Initialize(ctx context.Context, root string, args []string) error {
+
+	root = root + "/inflections.yml"
+
+	content := `
+	{
+	  "singular": "plural"
+	}
+	`
+
+	if _, err := os.Stat(root); err == nil {
+
+		fmt.Println("inflections.yml file already exist")
+		return nil
+
+	} else if os.IsNotExist(err) {
+
+		// create file if it does not exist
+		file, err := os.Create(root)
+
+		if err != nil {
+			return (err)
+		}
+
+		_, err = os.OpenFile(root, os.O_RDWR, 0644)
+		if err != nil {
+			return (err)
+		}
+
+		_, err = file.WriteString(content)
+		if err != nil {
+			return (err)
+		}
+
+		file.Close()
+
+		return nil
+
+	} else {
```

Comment:
Else not needed given the return you have inside.

Reply:
This could could be out :)

Reply:
If I remove it, It'll ask me to add a return by the end of the function. That way, I don't cover the "Schrödinger's" option haha

Reply:
Got it. I just feel this code is very complicated (nesting). let me suggest something.